package.json

```diff
@@ -89,7 +89,8 @@
   },
   "resolutions": {
     "ansi-regex": "5.0.1",
-    "@mantine/core/type-fest": "4.20.1"
+    "@mantine/core/type-fest": "4.20.1",
+    "ws": "8.17.1"
```

Comment:
**💬 suggestion:** ‏be more specific when setting resolutions, because it could break some packages in the future (imagine we update a package that use a feature introduced by ws `8.18.0`, because of the resolution set to `8.17.1`, the feature won't be there and the package will break).

Now, only `puppeteer-core@20.9.0` is using an outdated version of `ws`, so you could do:
```suggestion
    "puppeteer-core@npm:20.9.0/ws": "8.17.1"
```

Reply:
Thanks for the explanation, I used suggested resolution ✔️